pkg/cmd/secvulnMaven.go

```diff
@@ -5,33 +5,319 @@
 package cmd
 
 import (
+	"encoding/xml"
 	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"sort"
+	"strings"
 
 	"github.com/spf13/cobra"
 )
 
 var (
-    secvulnMavenCmd = &cobra.Command{
+	secvulnMavenCmd = &cobra.Command{
 		Use:   "maven",
 		Short: "Generate psuedo maven project for security vulnerability scanning",
 		Long:  "Generate psuedo maven project for security vulnerability scanning",
 		Run:   secvulnMavenExecute,
 	}
 
-	secvulnMavenParentDir     string
-	secvulnMavenPomUrls    *[]string
+	secvulnMavenParentDir string
+	secvulnMavenPomUrls   *[]string
+	secvulnMavenPomRepos  *[]string
+
+	completedProjects []string
+	toDoProjects      []string
 )
 
 func init() {
 	secvulnMavenCmd.PersistentFlags().StringVar(&secvulnMavenParentDir, "parent", "", "Parent project directory")
 	secvulnMavenPomUrls = secvulnMavenCmd.PersistentFlags().StringArray("pom", nil, "Component Pom URLs")
+	secvulnMavenPomRepos = secvulnMavenCmd.PersistentFlags().StringArray("repo", nil, "Repos to look for Poms")
 
 	secvulnMavenCmd.MarkPersistentFlagRequired("parent")
 	secvulnMavenCmd.MarkPersistentFlagRequired("pom")
+	secvulnMavenCmd.MarkPersistentFlagRequired("repo")
 
 	secvulnCmd.AddCommand(secvulnMavenCmd)
 }
 
 func secvulnMavenExecute(cmd *cobra.Command, args []string) {
 	fmt.Printf("Galasa Build - Security Vulnerability Maven - version %v\n", rootCmd.Version)
+
+	for _, pom := range *secvulnMavenPomUrls {
+		startScanningPipeline(pom)
+	}
+
+	updateParent()
+	fmt.Printf("Security scanning project pom.xml updated\n")
+
+}
+
+func startScanningPipeline(mainPomUrl string) {
```

Comment:
not sure why you call it pipeline,    its not.    which is confusing.     POM is more accurate

Reply:
done